Solutions/Azure Firewall/SolutionMetadata.json

```diff
@@ -4,7 +4,7 @@
 	"firstPublishDate": "2022-05-23",
 	"providers": ["Microsoft"],
 	"categories": {
-		"domains" : ["Security - Threat Protection"],
+		"domains" : ["Security - Threat Protection","Security - Automation (SOAR)"],
```

Comment:
Hi Vedanshi,
Is there any reason for adding another domain category here?
Thanks